### PR TITLE
 Fix the link in scaling-deployments.md

### PR DESCRIPTION
**Pull request by bobcode99**
Update the links for scaledobject_types.go in keda 2.6

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.6/concepts/scaling-deployments.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The only constraint is that the target `Custom Resource` must define `/scale` [s
 
 This specification describes the `ScaledObject` Custom Resource definition which is used to define how KEDA should scale your application and what the triggers are. The `.spec.ScaleTargetRef` section holds the reference to the target resource, ie. `Deployment`, `StatefulSet` or `Custom Resource`. 
 
-[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/main/api/v1alpha1/scaledobject_types.go)
+[`scaledobject_types.go`](https://github.com/kedacore/keda/blob/main/apis/keda/v1alpha1/scaledjob_types.go)
```
Comment (tomkerkhove) [inline content/docs/2.6/concepts/scaling-deployments.md:34]: This does not seem correct, can you fix the file name please?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: content/docs/2.6/concepts/scaling-deployments.md (hunk 1 above)
Comment (zroubalik) [reply content/docs/2.6/concepts/scaling-deployments.md:34]: and this should be imho fixed in the other doc versions

Comment (bobcode99) [reply content/docs/2.6/concepts/scaling-deployments.md]: > This does not seem correct, can you fix the file name please?

fixed

https://github.com/kedacore/keda-docs/pull/737/commits/430a60d4371b44f8863f0e78cebedb9b9fc4845b

Comment (bobcode99) [reply content/docs/2.6/concepts/scaling-deployments.md]: > and this should be imho fixed in the other doc versions

fixed

https://github.com/kedacore/keda-docs/pull/737/commits/e1c6bd06bd821e25498cb4300ecf70532fe42af0

**Review by tomkerkhove — APPROVED** [review]
Thanks a ton!

**Review by JorTurFer — APPROVED** [review]

**Comment by JorTurFer** [comment]
Thanks! ❤️

**Pull request merged**